### PR TITLE
Add R recipes: crayon, purrr, rmysql, tibble

### PR DESCRIPTION
**Pull request by inodb**
- [x] I have read the guidelines above.
- [x] This PR adds a new recipe.
- [ ] This PR updates an existing recipe.
- [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by inodb** [comment]
@bgruening Added some more R recipes: crayon, purrr, rmysql and tibble. Again, pretty popular R packages for bioinf, similar to #2337.

File: recipes/r-purrr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  build:
+    - r-base
+    - r-bh
+    - r-rcpp
+    - r-dplyr >=0.4.3
+    - r-lazyeval >=0.2.0
+    - r-magrittr >=1.5
+    - posix               # [win]
+    - {{native}}toolchain # [win]
+    - gcc                 # [not win]
```
Comment (bgruening) [inline recipes/r-purrr/meta.yaml]: For compiled package we have currently some extra guidlines: https://github.com/bioconda/bioconda-recipes/blob/master/GUIDELINES.md#cc

File: recipes/r-tibble/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+# Suggests: testthat, withr, knitr (>= 1.5.32), rmarkdown, nycflights13, microbenchmark
+requirements:
+  build:
+    - r-base
+    - r-rcpp
+    - r-assertthat
+    - r-lazyeval >=0.1.10
+    - posix               # [win]
+    - {{native}}toolchain # [win]
```
Comment (bgruening) [inline recipes/r-tibble/meta.yaml]: This looks strange. Can we remove the windows specific bits as we do not support Windows and probably never will?

File: recipes/r-rmysql/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    - lib/
+
+# Suggests: testthat
+requirements:
+  build:
+    - mysqlclient
+    - r-base
+    - r-dbi >=0.4
+    - posix               # [win]
+    - {{native}}toolchain # [win]
```
Comment (bgruening) [inline recipes/r-rmysql/meta.yaml]: See above.

File: recipes/r-rmysql/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  # md5: 49448ba4863157652311cc5ea4fea3ea
+  # sha1: 3bcfbee008276084cbb37a2b453963c61176a322
+  # patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
```
Comment (bgruening) [inline recipes/r-rmysql/meta.yaml]: Also feel free to remove these comments and the above. Everything what is not needed :)

**Comment by inodb** [comment]
@bgruening Thanks for the quick review! There seems to be some problem with having a dependency on `dbi >= 0.4` and having R version `3.2.2` e.g.: https://travis-ci.org/bioconda/bioconda-recipes/jobs/161172013#L503-L504. Any clue how to handle this? I just remove the `>= 0.4` from the `r-dbi` dep, but not sure how stable that is.

**Comment by bgruening** [comment]
Ideally we can use R3.3 and you can use this pinning. Lets see if I can work on this during the weekend

**Comment by inodb** [comment]
Thanks @bgruening !

**Comment by inodb** [comment]
@bgruening Did you have any success with R3.3? Thanks

**Comment by rvalieris** [comment]
these packages have been added to conda-forge.

https://github.com/conda-forge/r-crayon-feedstock
https://github.com/conda-forge/r-purrr-feedstock
https://github.com/conda-forge/r-rmysql-feedstock
https://github.com/conda-forge/r-tibble-feedstock

if you need to update any of them you can submit PRs there directly.

**Pull request closed without merging**